### PR TITLE
create new branch with simplified README and introduction

### PR DESCRIPTION
**Pull request by jlnav**
Now it'll be easier to discuss changes to the README and rtd splash-page independent of other changes I'm making.

This doesn't have to come in for this release, maybe for 1.0

### CODE REVIEW

**Review by jmlarson1 — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -25,240 +25,51 @@
...
+libEnsemble: A complete toolkit for dynamic ensembles of calculations
+=====================================================================
 
-libEnsemble is a Python_ toolkit for coordinating workflows of asynchronous and dynamic ensembles of calculations.
+Adaptive, portable, and scalable software to connect "deciders" to experiments or simulations.
 
-libEnsemble helps users take advantage of massively parallel resources to solve design,
-decision, and inference problems and expands the class of problems that can benefit from
-increased parallelism.
+• **Adaptive ensembles**: Generate parallel tasks on-the-fly based on previous computations.
```
Comment (shuds13) [inline README.rst:34]: Heres my effort:

- **Dynamic ensembles**: Generate new tasks on-the-fly based on previous computations.
- **Extreme scaling**: Run identical scripts on or across_ laptops, clusters, and leadership-class machines.
- **Dynamic resource management**: Dynamically assign and reassign CPU/GPU resources to tasks.
- **Application monitoring**: Ensemble members can run, monitor, and cancel apps.
- **Heterogeneous resources** Automatic detection and correct assignment on Nvidia, AMD and Intel GPUs.
- **Low start-up cost**: No additional background services or processes required.

Thoughts?

Comment (shuds13) [reply README.rst:34]: In Dynamic resources

we have dropped: "partitions of any size" (e.g. nodes) - which to my mind made clear it included nodes, but I'm guessing thats obvious already

File: README.rst [hunk 2]
```diff
@@ -271,102 +82,14 @@ Resources
     doi     = {10.1109/tpds.2021.3082815}
   }
 
-**Example Compatible Packages**
```
Comment (shuds13) [inline README.rst:274]: As we have community examples, we should prob have a link to that.

File: README.rst [hunk 3]
```diff
@@ -271,102 +82,14 @@ Resources
...
-- Tasmanian_ Toolkit for Adaptive Stochastic Modeling and Non-Intrusive ApproximatioN
-- VTMOP_ Fortran package for large-scale multiobjective multidisciplinary design optimization
-
-libEnsemble has also been used to coordinate many computationally expensive
-simulations. Select examples include:
-
-- OPAL_ Object Oriented Parallel Accelerator Library. (See this `IPAC manuscript`_.)
-- WarpX_ Advanced electromagnetic particle-in-cell code. (See example `WarpX + libE scripts`_.)
-
-See a complete list of `example user scripts`_.
```
[the remark this answers is not in the document]
Comment (shuds13) [reply README.rst:307]: Also we should check pdf still works okay.

File: README.rst (hunk 1 above)
Comment (shuds13) [reply README.rst:34]: Coordinated data-flow between tasks - does this mean via the manager? I'm not sure about this one.

Comment (jlnav) [reply README.rst:34]: > Here's my effort:

Looks good! But I wonder if there's a way of combining the Dynamic resource management and Heterogeneous resources sections?

> In Dynamic resources

Yeah, that's what I was thinking. Plus "nodes" is covered in my mind by "extreme scaling".

> Coordinated data-flow between tasks - does this mean via the manager? I'm not sure about this one.

I'm trying to emphasize that the tasks don't exist independently, each in a "bubble" (which is the case for most multi-task workflow systems). They can communicate state with the manager and each other

File: README.rst (hunk 2 above)
Comment (jlnav) [reply README.rst:274]: Community Examples is still linked to on line 67 on the right

File: README.rst (hunk 1 above)
Comment (jlnav) [reply README.rst:34]: Oh yeah, I'm also trying to avoid redundant adjectives whenever possible.

**Review by shuds13 — APPROVED** [review]

**Pull request merged**